spec/models/comment_spec.rb

```diff
@@ -1,5 +1,38 @@
 require 'rails_helper'
 
 RSpec.describe Comment, type: :model do
```

Comment:
There are still some tests failing. Kindly make sure all the tests are passing.
![tests](https://user-images.githubusercontent.com/35504149/167266150-05cff473-4ceb-4933-9a10-8633daa4c46d.png)

Reply:
Hey @SajjadAhmad14. Thanks for your feedback. Please pull the branch again, I might have fixed the specs after you cloned the repo already.